cobl2/scripts/initializedb.py

```diff
@@ -92,6 +93,20 @@ def clean_md(t):
             lines.append(line)
         return '\n'.join(lines)
 
+    re_links = re.compile(r'\[(?P<label>[^\]]+?)\]\((?P<type>.+?)-(?P<id>\d+)\)')
+    link_map = {
+        'cog': '/cognatesets/',
+        'lex': '/values/',
+        'src': '/sources/',
+    }
+    def parse_links(m):
+        try:
+            return '<a href="%s%s">%s</a>' % (
+                link_map[m.group('type')], m.group('id'), m.group('label'))
+        except:
```

Comment:
The error we are expecting here is `KeyError`, right? If so, this should be made explicit: `except KeyError:`.

Reply:
yes - I'll add it